tools/grunt-tasks/minifyNames.js

```diff
@@ -1395,14 +1395,11 @@ function minifyNamesFn(grunt) {
                         // remove unused imports
                         removeUnusedImports(parsedFile);
             
-                        if (parsedFile.changed) {
-            
-                            var src = formatFile(parsedFile, theOptions.maxImportWidth);
-                            if (orgSrc !== src) {
-                                if (!theOptions.testOnly) {
-                                    // Rewrite the file
-                                    fs.writeFileSync(inputFile + theOptions.ext, src);
-                                }
+                        var src = formatFile(parsedFile, theOptions.maxImportWidth);
```

Comment:
This is the only file in this PR that was manually changed, all other changes are automatic and generated by this script.